packages/core/src/schema/models/documentLogs.ts

```diff
@@ -17,12 +24,13 @@ export const documentLogs = latitudeSchema.table(
         onUpdate: 'cascade',
       }),
     resolvedContent: text('resolved_content').notNull(),
-    parameters: json('parameters').notNull(),
+    parameters: jsonb('parameters').$type<Record<string, unknown>>().notNull(),
```

Comment:
always use jsonb (better performance and less disk space)

Reply:
Not sure how json is stored tbh, jsonb is stored in binary format and is generally understood to be faster and occupy less space